instance/models/instance.py

```diff
@@ -373,8 +398,7 @@ def set_to_branch_tip(self, branch_name=None, ref_type=None, commit=True):
 
             # Update the hash in the instance title if it is present there
             # TODO: Find a better way to handle this - include the hash dynamically?
-            # TODO: Figure out why the warnings aren't suppressed despite the fact that it's a mixin
-            if self.name and old_commit_short_id: #pylint: disable=access-member-before-definition
+            if self.name and old_commit_short_id:
```

Comment:
Just out of curiosity – any idea what changed so that we don't need to suppress this warning anymore?